OpenOversight/tests/conftest.py

```diff
@@ -48,6 +49,10 @@
     "Chief",
 ]
 
+DEPARTMENT_NAME = "Springfield Police Department"
+DEPARTMENT_TWO_NAME = "Chicago Police Department"
+DEPARTMENT_EMPTY_NAME = "Empty Police Department"
```

Comment:
When I see "empty" in the name, I presume it's an empty string.

Reply:
Good point. Changed it to `DEPARTMENT_WITHOUT_OFFICERS`.